homeassistant/components/mobile_app/__init__.py

```diff
@@ -79,9 +85,15 @@
     webhook_register(hass, DOMAIN, registration_name, webhook_id,
                      handle_webhook)
 
+    hass.async_create_task(
+        hass.config_entries.async_forward_entry_setup(entry,
+                                                      DATA_BINARY_SENSOR))
+    hass.async_create_task(
+        hass.config_entries.async_forward_entry_setup(entry, DATA_SENSOR))
+
     if ATTR_APP_COMPONENT in registration:
-        load_platform(hass, registration[ATTR_APP_COMPONENT], DOMAIN, {},
-                      {DOMAIN: {}})
+        await async_load_platform(hass, registration[ATTR_APP_COMPONENT],
```

Comment:
What platform are we loading here? Why can't we just forward entry setup as we do above?

Reply:
This is a external component which may or may not exist and we shouldn't expect it to support config entries. External components allow for adding custom webhook handlers.

Reply:
Removed for later PR